src-v2/Not/Not.Blazor/TM/NotTimer.razor

```diff
@@ -11,23 +12,29 @@
     [Parameter]
     public double StopTimerAtTime { get; set; } = default!;
 
+    const int TimerRefreshInMilliseconds = 1000;
+
     private Timer _timer = default!;
     private MudTheme _theme = new MudTheme();
     private TimeSpan _time = default!;
     private string _displayTime = default!;
     private string _color = default!;
 
+    protected override void OnInitialized()
+    {
+        _timer = new Timer(TimerRefreshInMilliseconds);
+    }
+
     protected override void OnParametersSet()
     {
         _time = TimeSpan.Parse(StartTime);
         FormatTime();
-        _timer = new Timer(1000);
         _timer.Elapsed += OnTimerElapsed;
         _timer.AutoReset = (_time > TimeSpan.FromMinutes(StopTimerAtTime));
         _timer.Enabled = _time > TimeSpan.FromMinutes(StopTimerAtTime);
     }
 
-    private void OnTimerElapsed(object? sender, System.Timers.ElapsedEventArgs e)
+    private async void OnTimerElapsed(object? sender, System.Timers.ElapsedEventArgs e)
     {
         _time = _time.Subtract(TimeSpan.FromSeconds(1));
```

Comment:
Why not simply use `-` operator?

Reply:
Furthermore `TimeSpan` value can be initialized once, not on every elapsed handler

Reply:
Substract seems clearer in this case.

Reply:
No. We're using operators when they are defined. Otherwise we should start using `5.LessThan(10)`, `var a = 5.Minuts(3)` and so on. It's not clearer, it's excessively verbose.